src/components/Members/Members.js

```diff
@@ -52,14 +52,17 @@ class Members extends React.Component {
                     </form>
                 </section>
                 <div className="container card memberWrap">
-                    <MembersCard />
+                    <MembersCard members={this.props.users.instructors} />
```

Comment:
Is this only passing in instructors? Do we want it to be all members?

Reply:
I updated this in my most recent PR!